sources/packages/web/src/composables/institution/useInstitutionAuth.ts

```diff
@@ -1,34 +1,60 @@
 import {
   InstitutionUserAuthRolesAndLocation,
   InstitutionUserRoles,
+  UserStateForStore,
 } from "@/types";
 import { computed } from "vue";
-import { useStore } from "vuex";
+import { Store, useStore } from "vuex";
 import { useAuth } from "..";
 
-export function useInstitutionAuth() {
-  const store = useStore();
+export function useInstitutionAuth(rootStore?: Store<any>) {
+  const store = rootStore ?? useStore();
```

Comment:
why `rootStore`

Reply:
As discussed when the composable NOT used from vue files, useStore() is undefined and we need to pass the store to the composable. I just followed the pattern of useStudentStore() and didn't investigate why it was done in that way.